src/ui/editortabwidget.cpp

```diff
@@ -24,6 +24,7 @@ EditorTabWidget::EditorTabWidget(QWidget *parent) :
 
 EditorTabWidget::~EditorTabWidget()
 {
+    this->blockSignals(true);
```

Comment:
We don't need to receive signals from this when its destructor is being called.... it causes issues regarding pointer validity(somehow)... and causes a crash.